site/web/static/img/logo_email_notification.svg

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Use a PNG image here, don't expect email clients to be able to handle SVG images (outlook most certainly can't).